README.md

```diff
@@ -1,54 +1,88 @@
-# Kwil DB
+# KwilDB
 
 The database for Web3.
 
+![banner](./assets/kwil-banner.jpeg) 
+
+![Github-License](https://img.shields.io/badge/license-Apache%202.0-green) ![Release](https://img.shields.io/github/v/release/kwilteam/kwil-db) ![Build-Status](https://github.com/kwilteam/kwil-db/actions/workflows/ci.yaml/badge.svg) ![Go-Version](https://img.shields.io/github/go-mod/go-version/kwilteam/kwil-db) ![GoDoc](https://godoc.org/github.com/kwilteam/kwil-db?status.svg)
```

Comment:
All good, but markdown is unphased by newlines unless there's a blank line separating, so we're free to put these on separate lines in .md and still have them be on the same line when rendered.  No need to change, just a note.